broken-site-reporting/tests.json

```diff
@@ -34,7 +34,7 @@
                 "expectReportURLPrefix": "https://improving.duckduckgo.com/t/epbf",
                 "expectReportURLParams": [
                     {"name": "category", "value": "paywall"},
-                    {"name": "siteUrl", "value": "https%3A%2F%2Fsub.example.test%2F"},
+                    {"name": "siteUrl", "value": "https%3A%2F%2Fsub.example.test%2Fpath%2Fto%2Fthing.html"},
```

Comment:
this is unrelated to https changes, but I found it while updating tests, but full path of the site should be included in the report